### DISCUSSION TITLE
Resume Guidelines

### DISCUSSION THREAD

**Discussion by LovleenKaur-tal** [snigi-gupta/AttendingUniversityAtBuffalo#25] [General]
Hi Snighda, 

Thank you so much for assisting the graduates by providing best repository that can answer most of the queries of students. 
I would like know if you can upload the guidelines for building a resume. Looking forward to it. 

Best 
Lovleen Kaur

**Comment by snigi-gupta** [comment]
Hi @LovleenKaur-tal ,

Yes, I am working on it. I am trying to publish it by end of June. Keep a check on the space. :)

**Reply by LovleenKaur-tal** [reply]
Thanks a Lot Snigdha
